civis/io/_tables.py

```diff
@@ -617,8 +617,7 @@ def dataframe_to_civis(df, database, table, api_key=None, client=None,
                        max_errors=None, existing_table_rows="fail",
                        diststyle=None, distkey=None,
                        sortkey1=None, sortkey2=None,
-                       headers=None, credential_id=None,
```

Comment:
Rather than removing the `headers` parameter completely, you should leave it as an option in the function signature, and add a call to `warnings.warn`, something like 
```python
if headers is not None:
    warnings.warn('The "headers" parameter has been deprecated and will be removed in a future version of the API client. Setting it has no effect.', FutureWarning)
```
That way users won't have their code break if they'd been setting `headers` in this function. (Hopefully they weren't setting it to anything other than `True`, or their imports would have broken.) We can remove the parameter when we get to a v2 release.

Reply:
Done